io.github.androidvirtualpen.virtualpen.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "io.github.androidvirtualpen.virtualpen",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "virtual-pen-linux-host",
+    "finish-args": [
+	"--socket=wayland",
+	"--socket=fallback-x11",
+	"--share=ipc",
+	"--filesystem=home",
```

Comment:
why `home` filesystem access?

Reply:
where exactly? It should just use the XDG standard locations

Reply:
I don't see this point answered.

Reply:
I removed the --filesystem=home entry, the config location I was accessing was indeed within the XDG standard locations (explicitly tested without an no issue saving using settings).